src/pytest_inmanta_lsm/remote_orchestrator.py

```diff
@@ -117,7 +134,7 @@ def ensure_project(project_name: str) -> str:
             result = client.project_list()
             assert (
                 result.code == 200
-            ), f"Wrong reponse code while verifying project, got {result.code} (expected 200): \n{result}"
+            ), f"Wrong reponse code while verifying project, got {result.code} (expected 200): \n{result.result}"
```

Comment:
Not something you did, but while you are there, could you also fix the assertion message below? By adding the `.result` as you did here